docs/application/native/guides/text-input/capi-ui-gesture.md

```diff
@@ -0,0 +1,247 @@
+# Gesture
+
+`Gesture` is a movement or position of the hand, body, head, and face, and it expresses emotion, opinion, and so on.
+
+As there are many gesture types, so there are various methods to recognize gestures on the device.
+For example, hand gestures are recognized, based on gyro sensor data. And, head movements can be detected with a visual camera and video frames.
+According to the targeted gesture, the device and the input sensor, the recognition method can be changed.
+
+In the current Gesture, only wearable device is targeted and only hand gesture is recognized from input sensor data.
+
+The main features of the Gesture API include:
+
+- Getting the supported gesture type
+
+  You can [get the supported gesture type](#info_gesture) to be recognized.
+
+- Setting gesture options
+
+  You can [set options](#set_option) such as a hand gesture type and option.
+
+- Preparing the Gesture service for use
+
+  You can [connect the background Gesture daemon](#prepare) to be able to operate the gesture.
+
+
+<a name="basic_gesture"></a>
+## Basic Gesture processes
+
+There are processes you should follow in order to get the recognized gesture. Using Gesture, you can:
+
+1. [Create a handle](#create_destroy) and register a error callback function.
+  - Create a Gesture handle, which is used for distinguishing your application from other applications that are also using Gesture.
+  - To get notifications about errors, [register callback functions](#set).
+2. [Check the supported gesture type](#info_gesture).
+  - Check the gesture type which can be recognized by the gesture service.
+3. [Set an option](#set_option) for gesture recognition.
+  - Set an option to select always-on mode or not.
+4. [Start the gesture recognition](#start_stop) and [register a callback function](#callback) for recognition results.
+  - Start recognizing the gesture.
+  - To get the recognition result.
+5. Stop the gesture recognition.
+  - Stop recognizing the gesture.
+6. Destroy the handle.
+
+
+## Prerequisites
+
+To enable your application to use the Gesture functionality in [wearable](../../api/wearable/latest/group__CAPI__UIX__GESTURE__MODULE.html) applications, include the `<gesture.h>` header file in your application:
+
+    ```c
+    #include <gesture.h>
+    ```
+
+
+<a name="create_destroy"></a>
+## Create and Destroy handle
```

Comment:
Global: Please use sentence case

## Create and destroy handle

Reply:
Done.